src/webui/www/private/preferences_content.html

```diff
@@ -559,7 +559,7 @@
     getWatchedFolders = function() {
         var nb_folders = $("watched_folders_tab").getChildren("tbody")[0].getChildren("tr").length;
         var folders = new Hash();
-        for (var i = 0; i < nb_folders; i++) {
+        for (var i = 0; i < nb_folders; ++i) {
```

Comment:
Does it make any sence in JavaScript?

Reply:
It makes as much (or as little) as in C++.
This is mainly for coding style unification, I can revert if any one insist.

Reply:
>This is mainly for coding style unification

Ok, that's a pretty good reason.

Reply:
No objections from me, since we're changing all instances.